### ISSUE TITLE
user-sync.2.6.4 fails with Error loading Python lib '/tmp/_MEILQgGGA/libpython3.6m.so.1.0 on ubuntu

### ISSUE THREAD

**Issue by mictsi** [adobe-apiplatform/user-sync.py#732]
**Description**
Tried to update to user-sync to v2.6.4 from 2.6.2 on an ubuntu 18.04 and it fails with the following error when running the binary.
```
`GLIBC_2.29'[21364] Error loading Python lib '/tmp/_MEILQgGGA/libpython3.6m.so.1.0': dlopen: /lib/x86_64-linux-gnu/libm.so.6: version `GLIBC_2.29' not found (required by /tmp/_MEILQgGGA/libpython3.6m.so.1.0)Error loading Python lib '/tmp/_MEILQgGGA/libpython3.6m.so.1.0
```

I tried to run both ubuntu binaries from the release page,  user-sync-v2.6.4-noext-ubuntu.tar.gz and user-sync-v2.6.4-ubuntu.tar.gz. 

2.6.2 runs fine without errors. 

**Steps to reproduce**
run the binary with the same configuration as 2.6.2 with no command line options and the errors occured. 

**Expected behavior**
I would expect no error when running

**Environment**
 - OS type and version: Ubuntu 18.04.6 LTS

**Comment by adorton-adobe** [comment]
Looks like we need a separate build for 18.04. I'll add it for 2.6.5, which should be released this week.

**Comment by adorton-adobe** [comment]
@mictsi - We've released 2.6.5 which includes a Bionic (18.04) build. Please let me know if you have any issues.

**Comment by mictsi** [comment]
Hi, tested the bionic version and it works fine. Thanks for the quick fix.
